src/utils/aspect-ratio-support.tsx

```diff
@@ -103,7 +108,11 @@ export const AspectRatioElement = (props: AspectRatioElementProps): JSX.Element
         })();
 
         return (
-            <div style={{width, height, paddingTop}} className={styles.wrapper}>
+            <div
+                style={{width, height, paddingTop}}
+                className={styles.wrapper}
+                {...getPrefixedDataAttributes(props.dataAttributes)}
```

Comment:
set the second parameter to the component name:

`getPrefixedDataAttributes(props.dataAttributes, 'AspectRatioElement`)`

btw, it is kinda weird that a component is named as "Element", the component could be renamed to just "AspectRatio"

Reply:
fixed and changed name of AspectRatioElement to AspectRatioContainer